sellercenter-category/pom.xml

```diff
@@ -52,15 +52,30 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.5</version>
+            <artifactId>mockito-core</artifactId>
```

Comment:
mockito-all is abandoned since 5 years and during run it outputs lot of warnings